requirements.txt

```diff
@@ -123,7 +123,7 @@ django-rest-auth==0.9.5
     # via -r ./requirements.in
 django-storages==1.13.2
     # via -r ./requirements.in
-django-test-plus==1.4.0
+django-test-plus==2.2.1
```

Comment:
Sure, I can do that. I didn't do that because:

1- I figured we didn't want to pin the dep, since in general we'd want latest?
2- I didn't wanted to update the whole requirements.txt to avoid mixing concerns in this PR.

Given the above, if your advice is still to pin it, I'll happily do it!